### ISSUE TITLE
nightly builds are failing

### ISSUE THREAD

**Issue by bgruening** [bioconda/bioconda-recipes#1516]
The nightly builds are failing because of a missing expat library.

https://travis-ci.org/bioconda/bioconda-tests/jobs/128760945

This one is already included in conda-forge. We should focus on solving the conda-forge union imho and depend on this channel. It will make our life easier.

Any idea how to proceed here before we can depend on conda-forge?

**Comment by johanneskoester** [comment]
Indeed, some nightlies are failing at the moment. The good news is that the vast majority of the packages works ok.

Expat is also available in the main channel, but the graphviz package is not including nor pinning it. I think, at the moment, we should focus on getting all packages to work in the nightly tests, when using the r and the default channel. Once that is achieved, we can think about adding conda-forge. Also, we might want to wait until the dust with gcc issues on conda-forge has settled. Finally, there are improvements for pinning underway in conda-build.
Bottom line: my feeling is that we currently have better stability when not relying on conda-forge (because it introduces another point of failure). This might change at some point. I still like the idea of depending on it.

The latest nightly build had some additional issues with downloading from anaconda.org, but I hope these will go away tomorrow. I would suggest that we start fixing the packages as soon as possible.

**Comment by bgruening** [comment]
Afaik nightly builds are currently not working and maintained. We have some ideas to enhance testing and I will close this for now.
